Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PG_VERSION=14.4
+ARG PG_VERSION=14.8
```

Comment:
It was released as `14.6`, but seems to have actually remained at `14.4`. Maybe that was intentional though.